packages/ove-app-videos/README.md

```diff
@@ -0,0 +1,37 @@
+# Videos App
+
+This app supports playing videos using the OVE framework. There are two players available, `HTML5`, which is a generic player for any video, and `YouTube` which is based on the [YouTube IFrame Player API](https://developers.google.com/youtube/iframe_api_reference).
+
+## Application State
+
+The state of this app has a format similar to what is provided below:
+
+```json
+{
+    "url": "http://www.youtube.com/embed/XY3NP4JHXZ4"
+}
+```
+
+The player is selected automatically based on the URL: Any YouTube URL uses the YouTube player and all other URLs use the HTML5 player.
+
+## Loading the App
+
+A video can be loaded using the OVE APIs:
+
+Windows:
+
+```sh
+curl --header "Content-Type: application/json" --request POST --data "{\"h\": 500, \"app\": {\"url\": \"http://OVE_APP_VIDEOS_HOST:PORT\", \"states\": {\"load\": {\"url\": \"http://www.youtube.com/embed/XY3NP4JHXZ4\"}}}, \"space\": \"LocalNine\", \"w\": 500, \"y\": 0, \"x\": 0}" http://OVE_CORE_HOST:PORT/section
+```
+
+Linux/Mac:
+
+```sh
+curl --header "Content-Type: application/json" --request POST --data '{"h": 500, "app": {"url": "http://OVE_APP_VIDEOS_HOST:PORT","states": {"load": {"url": "http://www.youtube.com/embed/XY3NP4JHXZ4"}}}, "space": "LocalNine", "w": 500, "y": 0, "x": 0}' http://OVE_CORE_HOST:PORT/section
+```
+
+## Controlling the App
+
+Once the app is loaded, it can be controlled via the URL `http://OVE_APP_VIDEOS_HOST:PORT/control.html?oveSectionId=0`.
```

Comment:
List the operations that can be performed using the controller.

Reply:
Explained above. The controller doesn't expose any functionality as yet and the docs will be updated when we have such.

Reply:
in which case i think it is important that the curl command to at least start playing the video is documented here.

Reply:
@davidbirchwork, ok, I will also fix the same in the audio app, which even doesn't provide a proper Swagger URL to start with, :).